core/src/main/java/greencity/controller/SuperAdminController.java

```diff
@@ -190,76 +192,79 @@ public SuperAdminController(SuperAdminService superAdminService) {
     /**
      * Get all info about locations, and min amount of bag for locations.
      * 
-     * @return {@link GetLocationTranslationDto}
+     * @return {@link FindInfoAboutLocationDto}
      * @author Vadym Makitra
      */
     @ApiOperation(value = "Get info about location and min amount of bag for this location")
     @ApiResponses(value = {
-        @ApiResponse(code = 200, message = HttpStatuses.OK, response = GetLocationTranslationDto.class),
+        @ApiResponse(code = 200, message = HttpStatuses.OK, response = FindInfoAboutLocationDto.class),
         @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
         @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN)
     })
     @GetMapping("/getLocations")
-    public ResponseEntity<List<GetLocationTranslationDto>> getLocations() {
+    public ResponseEntity<List<FindInfoAboutLocationDto>> getLocations() {
         return ResponseEntity.status(HttpStatus.OK).body(superAdminService.getAllLocation());
     }
 
     /**
      * Create new Location.
      * 
      * @param dto {@link LocationCreateDto}
-     * @return {@link GetLocationTranslationDto}
+     * @return {@link FindInfoAboutLocationDto}
```

Comment:
`FindInfoAboutLocationDto` is not the best name for the class. Give it `LocationInfoDto` or something.